packages/workspace/src/command-line/lint.ts

```diff
@@ -6,10 +6,16 @@ import { WorkspaceIntegrityChecks } from './workspace-integrity-checks';
 import { readWorkspaceFiles, workspaceLayout } from '../core/file-utils';
 import { output } from '../utilities/output';
 import * as path from 'path';
+import { existsSync, readFileSync } from 'fs';
+import { assertJestConfigValidity } from '../core/assert-jest-configuration';
 
 export function workspaceLint() {
   const graph = onlyWorkspaceProjects(createProjectGraph());
 
+  if (existsSync('jest.config.js')) {
+    assertJestConfigValidity(graph, readFileSync('jest.config.js').toString());
```

Comment:
Shouldn't `jest` check the validity of the `jest` config? 👀 

The issue you're trying to solve is that our generator can't modify the `jest.config.js` file right?

I think it's alright to warn if it's unable to modify it.

The user could write their `jest.config.js` by hand. I don't think `nx workspace-lint` should error if it doesn't comply with our generator. 🤔

Reply:
Hey @FrozenPandaz 👋

The issue we're trying to solve here is actually to lint to ensure that our root jest config (if it exists) is in sync with our workspace graph (in a similar way we lint for nx.json and workspace.json being out of sync).

My understanding is that the root `jest.presets.js` file is there for any custom config that a single project would inherit from, and this root `jest.config.js` file now exists only to help IDE's run jest tests (with ability to have those in-line run/debug buttons).

![Screen Shot 2021-04-14 at 1 06 35 PM](https://user-images.githubusercontent.com/3788405/114888889-7a87cf80-9dbe-11eb-8fe0-fab26179fdac.png)

Is there a better way to inform our users about their jest config being out of sync? (cc @jaysoo)